### PR TITLE
use absolute URL for external media

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

File: Media/DisplayMedia/Strategies/AbstractStrategy.php [hunk 1]
```diff
@@ -13,6 +14,18 @@
...
 {
     protected $router;
+    protected $request;
+    protected $mediaDomain;
+
+    /**
+     * @param RequestStack $requestStack
+     * @param string       $mediaDomain
+     */
+    public function __construct(RequestStack $requestStack, $mediaDomain)
```
Comment (nicolasThal) [inline Media/DisplayMedia/Strategies/AbstractStrategy.php]: you could put a base value at '' for instance

**Comment by itkg-rboessel** [comment]
@itkg-ngilain @itkg-nanne @itkg-alavieille

**Comment by itkg-rboessel** [comment]
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/3

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**